ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/addressbook/v1/addressbook/README.md

```diff
@@ -0,0 +1,12 @@
+# Address Book (v1)
+
+The Address Book is a React component providing CRUD operations on the address book of a shopper.
+
+## Features:
+* List the address book entries
+* Add an address book entry
+* Change the address on an existing entry
+* Remove an address from the address book
+
+## Implementation
+This AEM component only renders a `div` element acting as a container for the [AddressBook](../../../../../../../../../../../../../react-components/src/components/AddressBook) React component.  
```

Comment:
Might be easier to set an absolute link to `/react-components/src/components/AddressBook` no?

Reply:
Yes, it's true, I just blindly followed the readme for the Minicart :)